js/geo/lng_lat.js

```diff
@@ -18,6 +18,9 @@ var wrap = require('../util/util').wrap;
  * @param {number} lat Latitude, measured in degrees.
  * @example
  * var ll = new mapboxgl.LngLat(-73.9749, 40.7736);
+ * @see [Get coordinates of the mouse pointer](https://www.mapbox.com/mapbox-gl-js/example/mouse-position/)
+  * @see [Display a popup](https://www.mapbox.com/mapbox-gl-js/example/popup/)
+  * @see [Highlight features within a bounding box](https://www.mapbox.com/mapbox-gl-js/example/using-box-queryrenderedfeatures/)
```

Comment:
Hmm... I don't see either fix on the Github diff. Are you sure you pushed the changes?